lib/deppack/modules.js

```diff
@@ -20,6 +20,14 @@ const getModuleRootName = path => {
   return split[index + 1];
 };
 
+const getModuleFullRootName = path => {
+  const split = path.split(sysPath.sep);
+  const indexS = split.indexOf('node_modules');
+  const indexE = split.lastIndexOf('node_modules');
+  return split.slice(indexS + 1, indexE + 2).join('/');
```

Comment:
This is going to be a *module name*, not some kind of path. Module names are separated with `/`